.github/workflows/sysid_test.yml

```diff
@@ -27,8 +27,8 @@ jobs:
       - name: Run subsystem level tests on parametric model
         run: Tools/parametric_model/test_parametric_model.sh
       - name: Run multirotor_model estimation for system level test using resource ulog
-        run: make estimate-model model=quadrotor_model plot=False
+        run: make estimate-model model=quadrotor_model log=resources/quadrotor_model.ulg plot=False
```

Comment:
We don't need the `log` flag as long as the ulog name matches the model name (`quadrotor_model`)

Reply:
yes that is true, i wanted to be explicit since i was thinking about changing the default command to use the csv. But if you have strong objections we can remove it.

Reply:
Since this is in the CI, I think it is better we remove it so that we test the make targets implicitly.(Or that was the intention of the naming rules)